### ISSUE TITLE
Effect of Domain Specific BatchNorm

### ISSUE THREAD

**Issue by tarun005** [kgl-prml/Contrastive-Adaptation-Network-for-Unsupervised-Domain-Adaptation#13]
Hi,

Do you know what is the effect of domain specific batch norm on the final accuracy? What would be the drop without it?
